paddle/go/cmd/master/master.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"net/http"
+	"net/rpc"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/PaddlePaddle/Paddle/paddle/go/master"
+	"github.com/wangkuiyi/recordio"
+)
+
+const (
+	taskTimeoutDur = 20 * time.Minute
+	taskTimeoutMax = 3
+)
+
+func main() {
+	port := flag.Int("p", 0, "port of the master server")
+	dataset := flag.String("d", "", "dataset: comma separated path to RecordIO files")
```

Comment:
@typhoonzero Done. Added support for glob pattern. But not supporting list of dirs, I think directory is too coarse, it's hard to specify what should happen when there is a non-recordIO format under that directory.
@wangkuiyi Done.